pkg/utilities/version/version.go

```diff
@@ -71,11 +71,9 @@ func LessThan(v1, v2 string) (bool, error) {
 }
 
 func parseVersion(v string) (int, int, int, error) {
```

Comment:
This really should be using https://github.com/kubernetes/apimachinery/blob/master/pkg/util/version/version.go

Reply:
Changed, updated `LessThan` as well.